config/routes.rb

```diff
@@ -2,4 +2,5 @@
 
 SolidusStripe::Engine.routes.draw do
   resources :payment_intents, only: :create
+  resources :webhooks, only: :create, format: false
```

Comment:
I think there could be conflicts here if we don't use some prefixes. `/webhooks` has high chances to be already taken.

Reply:
The generator will mount the routes within a `solidus_stripe` namespace, so I'll end up being `/solidus_stripe/webhooks`. Do you think that's enough? Or do you think people might end up mounting the engine on the root namespace?

Reply:
That's perfect. I was confused reading the PR description when you mention `POST /webhooks`. Solved!